templates/visuals/.api/v1.7.0/schema.capabilities.json

```diff
@@ -820,6 +824,92 @@
                         }
                     }
                 },
+                "fillRule" : {
+                    "type": "object",
+                    "description": "A color gradient that will be dispalyed to user as a minimum and maximum color pickers",
+                    "properties": {
```

Comment:
did you grab the schema for this from the client code?

Reply:
So, I wrote this schema by JSON settings that I took from enchanced scatter chart in core repo.
PowerBI-visuals-core\src\Clients\CustomVisuals\visuals\enhancedScatterChart\visual\enhancedScatterChart.ts
Also, I looked at classes here
https://github.com/Microsoft/PowerBI-visuals-tools/blob/master/templates/visuals/.api/v1.7.0/PowerBI-visuals.d.ts#L873